pages/lecteur/cnaf/beneficiaire/index.js

```diff
@@ -103,6 +118,26 @@ export default function Beneficiaire() {
           return accum
         }, {})
 
+      const devoirs = items
+        .map(i => i.getElementsByTagName('TOPPERSDRODEVORSA'))
+        .reduce((accum, value) => {
+          if (value[0]) {
+            accum[value[0].innerHTML] = (accum[value[0].innerHTML] || 0) + 1
+            accum["total"] = (accum["total"] || 0) + 1
+          }
+          return accum
+        }, {})
```

Comment:
```suggestion
        .reduce(processField, {})
```


avec par ailleurs
```js
const processField = (accum, value) => {
          if (value[0]) {
            accum[value[0].innerHTML] = (accum[value[0].innerHTML] || 0) + 1
            accum["total"] = (accum["total"] || 0) + 1
          }
          return accum
        }
```

pour pouvoir réutiliser cette fonction